components/header/menuDrawer/styles.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { createStyles } from "@solid/lit-prism-patterns";
+
+const styles = (theme) =>
+  createStyles(theme, ["icons", "headerBanner", "buttons"], {
+    icon: theme.icons.iconColor(theme.palette.primary.main),
+    "icon-close": {
+      ...theme.icons.iconColor(theme.palette.primary.text),
+      fontSize: theme.typography.h1.fontSize,
+    },
+    "menu-drawer-item__text": {
+      fontSize: theme.typography.body.fontSize,
+      fontWeight: theme.typography.fontWeightLight,
+    },
+    "menu-drawer-item__link": {
+      "text-decoration": "underline",
+    },
+    drawerContainer: {
+      borderRadius: "0px 0px 10px 10px",
```

Comment:
Lets use the borderRadius from the theme on this one, `theme.shape.borderRadius`. It corresponds to 4 by default, so you can you do `0 0 ${theme.shape.borderRadius * 2.5} ${theme.shape.borderRadius * 2.5}` instead.

Reply:
(But I think we might just want to use the default border radius - check with @KyraAssaad whether it should stay consistent with theme properties, or if we want it to be 10px.